site/priority.xml

```diff
@@ -27,127 +27,172 @@ limitations under the License.
-        and 255.
-      </li>
-    </ul>
-
-    <p>
-      Messages without a <code>priority</code> property are treated as
-      if their priority were 0. Messages with a priority which is
-      higher than the queue's maximum are treated as if they were
-      published with the maximum priority.
-    </p>
-
-    <h2>Resource Usage Considerations</h2>
-    <p>
-      If priority queues are desired, we recommend using between 1 and 10.
-      Currently using more priorities will consume more resources (Erlang processes).
-    </p>
-    
-    <h2>Interaction with consumers</h2>
-
-    <p>
-      It's important to understand how consumers work when working
-      with priority queues. By default, consumers may be sent a large
-      number of messages before they acknowledge any, limited only by
-      network backpressure.
-    </p>
-
-    <p>
-      So if such a hungry consumer connects to an empty queue to which
-      messages are subsequently published, the messages may not spend
-      any time at all waiting in the queue. In this case the priority
-      queue will not get any opportunity to prioritise them.
-    </p>
-
-    <p>
-      In most cases you will want to use the <code>basic.qos</code>
-      method in manual acknowledgement mode on your consumers, to
-      limit the number of messages that can be out for delivery at any
-      time and thus allow messages to be prioritised.
-    </p>
-
-    <h2>Interaction with other features</h2>
-
-    <p>
-      In general priority queues have all the features of standard
-      RabbitMQ queues: they support persistence, paging, mirroring,
-      and so on. There are a couple of interactions that should be
-      noted though:
-    </p>
-
-    <ul>
-      <li>
-        <a href="ttl.html">Messages which should expire</a> will still
-        only expire from the head of the queue. This means that unlike
-        with normal queues, even per-queue TTL can lead to expired
-        lower-priority messages getting stuck behind non-expired
-        higher priority ones. These messages will never be delivered,
-        but they will appear in queue statistics.
-      </li>
-      <li>
-        <a href="maxlength.html">Queues which have a max-length
-        set</a> will, as usual, drop messages from the head of the
-        queue to enforce the limit. This means that higher priority
-        messages might be dropped to make way for lower priority ones,
-        which might not be what you would expect.
-      </li>
-    </ul>
+      </p>
+
+      <p>
+        Publishers then can publish prioritised messages using the
+        <code>priority</code> field of
+        <code>basic.properties</code>. Larger numbers indicate higher
+        priority.
+      </p>
+
+      <p>
+        Because the on-disk format for a priority queue is different,
+        <b>priority queues can only be defined by arguments</b>, not
+        policies. Queues can never change the number of priorities they
```

Comment:
"not policies" is misleading as this is the first way we use to define priorities.

Reply:
That comment explains why policies are probably not a good idea in this case. This must be partially undone then 🤦‍♂️.